### PR TITLE
Fixed Layout and glitches

### PR DESCRIPTION
**Pull request by ibell8**
I made the layout for the user entries a bit better and fixed a couple of glitches with the addition and deletion of entries from our database.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -143,22 +143,11 @@ def settings():
 @app.route("/view_entries", methods=["GET", "POST"])
 @login_required
 def users_entries():
-    """The following data is fake entries that'll be deleted later
```
Comment (ibell8) [inline app.py:146]: This is no longer needed with the database functioning

File: app.py [hunk 2]
```diff
@@ -143,22 +143,11 @@ def settings():
...
-        entries made by our user from the database. For the time
-        being I'll just use the value from the entries
-        list that I made above
-    """
-    prev_entries = Entry.query.filter_by(current_user.id)
+    """When the user enters there entries page, we'll then use this function
+    to display all of their previous entries."""
+    # The following algorithm in the database functions file
+    prev_entries = get_entries(current_user.id)
+    print(prev_entries[0].timestamp)
```
Comment (ibell8) [inline app.py:150]: I forgot to remove this lol, my bad.

File: app.py [hunk 3]
```diff
@@ -177,20 +166,17 @@ def delete_entry():
         Later I'll replace with a database algorith"""
 
         index = int(flask.request.form["Delete"])
```
Comment (ibell8) [inline app.py:168]: index will be the integer "id" of the entry we want deleted.

File: app.py [hunk 4]
```diff
@@ -177,20 +166,17 @@ def delete_entry():
...
     return flask.redirect(flask.url_for("users_entries"))
 
 
 @app.route("/add_entry", methods=["GET", "POST"])
 def add():
     # new entry object information
     poster = current_user.id
-    title = flask.request.form("title")
-    contents = flask.request.form("entry")
+    title = flask.request.form["title"]
```
Comment (ibell8) [inline app.py:178]: I made this correction because, I needed for this to be in brackets not parenthesis.

File: database_functions.py [hunk 5]
```diff
@@ -7,7 +7,7 @@ def get_entries(user_id):
 
 
 def deleteEntry(entry_id):
-    entry = Entry.query.filter_by(id=entry_id)
+    entry = Entry.query.filter_by(id=entry_id).first()
```
Comment (ibell8) [inline database_functions.py:10]: I forgot to include ".first()" originally, but with this quick edit, I can pull the entry that we need to be deleted.

File: templates/entries.html [hunk 6]
```diff
@@ -14,18 +14,30 @@
...
-
+        
         {% for i in range(0,length) : %}
-            <div class="container p-2 m-5 bg-dark text-white">
-                <form action="/delete_entry" method='POST'>
-                    <button type='submit' name="Delete" value={{user_entries[i].id}}> X </button>
-                </form>
-                <h3>{{user_entries[i].title}}</h3>
-                <p>{{user_entries[i].content}}</p>
+           {% if (i%2)==0  %}
```
Comment (ibell8) [inline templates/entries.html:26]: Originally, the entries would be displayed 1 by 1 all the way down the page, depending on how long it was.
But now with this use of if statements, I was able to create a 2 by 2 layout all the way down the page, which is the correct layout according to our mockup design.

File: templates/home.html [hunk 7]
```diff
@@ -22,7 +22,7 @@ <h1>Hello, {{user}}</h1>
     </div>
 
     <div id="maincontent">
-        <form action="/addentry" method="POST">
+        <form action="/add_entry" method="POST">
```
Comment (ibell8) [inline templates/home.html:25]: Fixed a typo

**Comment by AtilG** [comment]
Looks good, sorry about the delay!

**Pull request merged**